posthog/test/base.py

```diff
@@ -692,3 +692,20 @@ def wrapped(self, *args, **kwargs):
                 self.assertQueryMatchesSnapshot(query)
 
     return wrapped
+
+
+def also_test_with_different_timezone(fn):
```

Comment:
:heart: This is awesome.

```suggestion
def also_test_with_different_timezone(fn):
    """
    Runs the test twice, including in a different timezone to catch timezone handling bugs.
    """
```